edge_cms_mail/edge_cms_mail/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for edge_cms_mail project.
+
+Generated by 'django-admin startproject' using Django 1.10.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '$$35*a6fj60t!1_n7#1qrvw^z39e*8iaf9$k&i#*v=^8f!6e(i'
```

Comment:
Use an environment variable if possible, or generate a "testing" key, or add the settings file to the .gitignore